### DISCUSSION TITLE
Request for guidance

### DISCUSSION THREAD

**Discussion by Duwimpuhwe** [mrdbourke/pytorch-deep-learning#492] [General]
Hi Daniel,

I'm in need of diving in image segmentation and object detection, any quick guidance (course links , video, docs, etc). If the framework is pytorch, that would be super.

Thanks

**Comment by mrdbourke** [comment]
Hi @Duwimpuhwe ,

For these, I'd suggest either the pre-trained models in PyTorch: 
* Segmentation - https://pytorch.org/vision/stable/models.html#semantic-segmentation
* Object detection - https://pytorch.org/vision/stable/models.html#object-detection-instance-segmentation-and-person-keypoint-detection 

Or the Ultralytics GitHub (one-stop shop for many PyTorch-based object detection + image segmentation models): https://github.com/ultralytics/ultralytics
